sktime/forecasting/base/_fh.py

```diff
@@ -158,7 +170,7 @@ def __new__(
     def __init__(
         self,
         values: Union[VALID_FORECASTING_HORIZON_TYPES] = None,
-        is_relative: bool = True,
+        is_relative: Optional[bool] = True,
     ):
         if is_relative is not None and not isinstance(is_relative, bool):
```

Comment:
is the case of `is_relative=None` correctly treated below?

Reply:
sorry, I don't know. I did not touch this part of the logic. is it relevant for my PR?